hoomd/write/gsd_burst.py

```diff
@@ -114,20 +114,28 @@ def _attach_hook(self):
                                               self.mode, self.write_at_start,
                                               sim.timestep)
 
-    def dump(self):
-        """Write all currently stored frames to the file and empties the buffer.
+    def dump(self, start=0, end=-1, clear_entire_buffer=True):
+        """Write stored frames in range to the file and empties the buffer.
 
         This method alllows for custom writing of frames at user specified
         conditions.
 
+        Args:
+            start (int): The first frame to write. Defaults to 0.
+            end (int): The last frame to write, must be positive integer.
```

Comment:
```suggestion
            end (int): The last frame to write.
```
Default is not a positive int